app/src/main/java/com/greenfox/kalendaryo/models/GoogleCalendar.java

```diff
@@ -3,14 +3,13 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 
-/**
- * Created by barba on 04/01/2018.
- */
+import java.io.Serializable;
 
-public class GoogleCalendar implements Parcelable {
+public class GoogleCalendar implements Parcelable, Serializable {
 
     String id;
     String summary;
+    String sharingOptions;
```

Comment:
type for sharingOption should be VisibilityOption enum, so it cannot take other value